scipy/stats/_distn_infrastructure.py

```diff
@@ -100,8 +100,10 @@
     (Differential) entropy of the RV.
 """
 _doc_fit = """\
-fit(data, %(shapes)s, loc=0, scale=1)
+fit(data)
     Parameter estimates for generic data.
+    See [`scipy.stats.rv_continuous.fit`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.rv_continuous.fit.html#scipy.stats.rv_continuous.fit) for detailed documentation of the
```

Comment:
The URL is appearing in the rendered documentation. (reST is not the same as markdown, unfortunately.) Also, ideally, I think we would want a [cross-reference](https://www.sphinx-doc.org/en/1.5/markup/inline.html) rather than explicit URL.

Reply:
My bad, I thought markdown could be used, converting to inline RST @mdhaber

Reply:
Oh what a dream it would be!